test/testdata/haddockComments/ConstFunction.hs

```diff
@@ -0,0 +1,4 @@
+module ConstFunction where
+
+f :: Int
+f = 233
```

Comment:
This one is not in use. Maybe you forgot to add a test for this?

Reply:
oops, I forgot this file...